src/components/text-editor/examples/text-editor-basic.tsx

```diff
@@ -1,4 +1,8 @@
-import { LimelTextEditorCustomEvent } from '@limetech/lime-elements';
+import {
+    LimelCheckboxCustomEvent,
+    LimelInputFieldCustomEvent,
```

Comment:
🤔 we never do this. Are these needed?

Reply:
I don't know why you think we never use the actual type for events…? I searched the lime-elements code for `CustomEvent` in VSCode, and while we do use `CustomEvent` in some examples, from the ten-or-so I checked it looks like we import the actual type in the majority of cases.

Is there any particular reason you would prefer that we didn't? 🙂

Reply:
No. Just checked other examples. If you say it's better, then we do